pkg/controller/tuned/tuned_controller.go

```diff
@@ -375,7 +375,7 @@ func (r *ReconcileTuned) Reconcile(request reconcile.Request) (reconcile.Result,
 		return reconcileResult, err
 	}
 
-	err = r.syncDaemonSet(tunedInstance)
+	err = r.syncDaemonSet(r.manifestFactory.TunedDaemonSetRHEL7, tunedInstance)
```

Comment:
This hardcoded version instance is also just for now, like line 32 above in `syncOperatorStatus()`?

Reply:
If the decision is made for several daemonsets to be used, then yes, there will be an additional r.manifestFactory.TunedDaemonSetRHEL8